src/pageLayout/constants/navigationHierarchy.ts

```diff
@@ -36,7 +36,7 @@ export interface NavItem {
 export const generateNavItems = (): NavItem[] => {
   const state: AppState = getStore().getState()
   const orgID = state?.resources?.orgs?.org?.id ?? ''
-  const quartzMe = state?.me?.quartzMe ?? null
+  const identity = state?.identity?.currentIdentity ?? null
```

Comment:
Erring on the safe side of including a guard here: we aren't using a selector to grab this, as this is a pure function that takes in a direct reference to state. 

It should be the case that the default state exists in the store already, but not worth the risk of removing the guards imho.